<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgMusicProperties</class>
 <widget class="QDialog" name="DlgMusicProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Music properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QRadioButton" name="SameMusicRD">
     <property name="text">
      <string>Continue playlist of precedent object</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="SameMusicPauseRD">
       <property name="text">
        <string>Continue playlist of precedent object but set to pause for this object</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="SameMusicReduceVolumeRD">
       <property name="text">
        <string>Continue playlist of precedent object but reduce volume to :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="VolumeReductionFactorCB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>7</number>
       </property>
       <item>
        <property name="text">
         <string notr="true">90</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">80</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">70</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">60</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">40</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">10</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true">%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="NewMusicRD">
     <property name="text">
      <string>Start a new playlist</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="AddMusicBt">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/view_bottom.png</normaloff>icons/view_bottom.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="RemoveMusicBt">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/view_remove.png</normaloff>icons/view_remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="UpMusicBt">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/up.png</normaloff>icons/up.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="DownMusicBt">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/down.png</normaloff>icons/down.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="EditMusicBt">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/aktion.png</normaloff>icons/aktion.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label1">
       <property name="text">
        <string>Total playlist duration : </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LabelDuration">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string notr="true">00:00:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="PlayListTable">
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>File</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Duration</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Start</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>End</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Volume</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fade In</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fade Out</string>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OKBT">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/agt_action_success.png</normaloff>icons/agt_action_success.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CloseBT">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/agt_action_fail.png</normaloff>icons/agt_action_fail.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SameMusicRD</tabstop>
  <tabstop>NewMusicRD</tabstop>
  <tabstop>AddMusicBt</tabstop>
  <tabstop>RemoveMusicBt</tabstop>
  <tabstop>UpMusicBt</tabstop>
  <tabstop>DownMusicBt</tabstop>
  <tabstop>EditMusicBt</tabstop>
  <tabstop>PlayListTable</tabstop>
  <tabstop>OKBT</tabstop>
  <tabstop>CloseBT</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
